color.go

```diff
@@ -1,102 +1,133 @@
 package color
 
 var (
-	Reset  = "\033[0m"
-	Bold   = "\033[1m"
-	Red    = "\033[31m"
-	Green  = "\033[32m"
-	Yellow = "\033[33m"
-	Blue   = "\033[34m"
-	Purple = "\033[35m"
-	Cyan   = "\033[36m"
-	Gray   = "\033[37m"
-	White  = "\033[97m"
+	Reset     = "\033[0m"
+	Bold      = "\033[1m"
+	Underline = "\033[4m"
+	Black     = "\033[30m"
+	Red       = "\033[31m"
+	Green     = "\033[32m"
+	Yellow    = "\033[33m"
+	Blue      = "\033[34m"
+	Purple    = "\033[35m"
+	Cyan      = "\033[36m"
+	Gray      = "\033[37m"
+	White     = "\033[97m"
 )
 
 // Ize is an alias for the Colorize function
 //
 // Example:
-//     println(color.Ize(color.Red, "This is red"))
+//
+//	println(color.Ize(color.Red, "This is red"))
 func Ize(color, s string) string {
 	return Colorize(color, s)
 }
 
 // Colorize wraps a given message in a given color.
 //
 // Example:
-//     println(color.Colorize(color.Red, "This is red"))
+//
+//	println(color.Colorize(color.Red, "This is red"))
 func Colorize(color, s string) string {
 	return color + s + Reset
 }
 
 // InBold wraps the given string s in Bold
 //
 // Example:
-//     println(color.InBold("This is bold"))
+//
+//	println(color.InBold("This is bold"))
 func InBold(s string) string {
 	return Colorize(Bold, s)
 }
 
+// InUnderline wraps the given string s in Underline
+//
+// Example:
+//
+//	println(color.InUnderline("This is underlined"))
+func InUnderline(s string) string {
```

Comment:
Personally, I would have preferred to call this function `Underline`, but that field is already taken by the variable so.. 😔